question/bank/viewcreator/templates/version_info.mustache

```diff
@@ -0,0 +1,42 @@
+{{!
+    This file is part of Moodle - http://moodle.org/
+
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Moodle is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    @template qbank_editquestion/version_info
+
+    Example context (json):
+    {
+        "addquestiondata": [
+          {
+                "version": "version html",
+                "extrahtml": "HTML for elements",
+          }
+        ]
+    }
+}}
+<div>
```

Comment:
can you a add an id to identify the selector if some one wants to change the style in their own themes.